src/helpers/github.ts

```diff
@@ -1,5 +1,22 @@
 import * as github from "@actions/github";
+import { GitHubIssue } from "../github-types";
 
 export function getGithubWorkflowRunUrl() {
   return `${github.context.payload.repository?.html_url}/actions/runs/${github.context.runId}`;
 }
+
+export function parsePriorityLabel(labels: GitHubIssue["labels"] | undefined) {
+  if (!labels) return 1;
+
+  for (const label of labels) {
+    const priorityLabel = typeof label === "string" ? label : (label.name ?? "");
```

Comment:
Why is empty string CI not working? I suppose its not set up in this repo @gentlementlegen rfc